### PR TITLE
12779 misc mhr flows ux updates

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#12779

*Description of changes:*
- Added custom HomeOwnersIcon and HomeLocationIcon components to be used as SVG v-icons
- Used in stepper
image

- UX tweak to submitting party name header

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/assets/svgs/HomeLocationIcon.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <svg
+  width="29px"
+  height="25px"
+  viewBox="0 0 29 25"
+  version="1.1"
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
```
Comment (owillborn) [inline ppr-ui/src/assets/svgs/HomeLocationIcon.vue]: Indentation

File: ppr-ui/src/main.ts [hunk 2]
```diff
@@ -30,6 +30,7 @@ import App from './App.vue'
 // Helpers
 import { fetchConfig, initLdClient, isSigningIn, isSigningOut } from '@/utils'
 import KeycloakService from 'sbc-common-components/src/services/keycloak.services'
+import { HomeLocationIcon, HomeOwnersIcon } from './assets/svgs/Index'
```
Comment (cameron-eyds) [inline ppr-ui/src/main.ts]: minor thing, but lower case `index` file name

**Review by cameron-eyds — APPROVED** [review]
Really cool implementation of the svgs. look forward to see these in action, well done 👍 
Is the hover smooth?

**Comment by owillborn** [comment]
> Really cool implementation of the svgs. look forward to see these in action, well done 👍 Is the hover smooth?
@cameron-eyds 
hover is oh so smooth

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Pull request merged**